### PR TITLE
Add optional parameter to point to different parent branch

### PR DESCRIPTION
**Pull request by pschroeder89**
In the event a repo is not using "master" as its parent branch, Branch mode currently fails. This PR allows the use of a parameter (`--parent-branch main`, `--parent-branch trunk`, `--parent-branch develop`, etc.) to specify a repository's parent branch name.
If no parent-branch option is supplied, the default is master.

### CODE REVIEW

**Comment by anapaulagomes** [comment]
Nice one, @pschroeder89! ✨
Could you please add a test for it? Also, an entry on our [CHANGELOG.md](https://github.com/anapaulagomes/pytest-picked/blob/dev/CHANGELOG.md) would be awesome. Thanks!

**Comment by pschroeder89** [comment]
> Nice one, @pschroeder89! ✨
> Could you please add a test for it? Also, an entry on our [CHANGELOG.md](https://github.com/anapaulagomes/pytest-picked/blob/dev/CHANGELOG.md) would be awesome. Thanks!

Done and done!

**Comment by anapaulagomes** [comment]
Hi @pschroeder89! I'll look it again next week.

File: tests/test_modes.py [hunk 1]
```diff
@@ -91,6 +91,15 @@ def test_should_return_command_that_list_all_changed_files(self):
             "master",
         ]
 
+    def test_should_return_command_that_list_all_changed_files_for_different_branch(
```
Comment (anapaulagomes) [inline tests/test_modes.py:94]: Nice one but I'm missing a test in `test_pytest_picked.py` too.

**Review by anapaulagomes — CHANGES_REQUESTED** [review]
Almost there. I tested locally and works fine. Well done! I'm only missing a test in `test_pytest_picked.py` (where we'll have faster feedback in case we break something in the future). Once this is done we're ready to go.

File: tests/test_modes.py (hunk 1 above)
Comment (pschroeder89) [reply tests/test_modes.py:94]: Done!

**Review by anapaulagomes — APPROVED** [review]
🎉

**Pull request merged**